README.md

```diff
@@ -1,4 +1,4 @@
-# TextWorld
+# TextWorld [![Build Status](https://travis-ci.org/Microsoft/TextWorld.svg?branch=master)](https://travis-ci.org/Microsoft/TextWorld) [![PyPI version](https://badge.fury.io/py/textworld.svg)](https://badge.fury.io/py/textworld)
```

Comment:
I think people usually put these on a new line under the title

Reply:
You are right. I'll move them.